src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -1272,6 +1272,7 @@ MavlinkReceiver::handle_message_set_actuator_control_target(mavlink_message_t *m
 
 		offboard_control_mode_s offboard_control_mode{};
 		offboard_control_mode.timestamp = hrt_absolute_time();
+		offboard_control_mode.actuator = true;
```

Comment:
It doesn't actually matter, but I try to set the timestamp as the last thing before publication, treating it like publication metadata.

Reply:
Thanks! Done!